src/lib.rs

```diff
@@ -174,18 +184,20 @@ extern crate quickcheck;
 extern crate serde_json;
 
 pub(crate) mod normal;
-pub use normal::*;
+pub use crate::normal::*;
 
 pub mod hop;
-pub use hop::HopSlotMap;
+pub use crate::hop::HopSlotMap;
 
 pub mod secondary;
-pub use secondary::SecondaryMap;
+pub use crate::secondary::SecondaryMap;
 
+#[cfg(feature = "std")]
 pub mod sparse_secondary;
-pub use sparse_secondary::SparseSecondaryMap;
+#[cfg(feature = "std")]
+pub use crate::sparse_secondary::SparseSecondaryMap;
```

Comment:
`HashMap` currently requires `std` because it assumes the default `RandomState`, which relies on OSRng. https://github.com/rust-lang/rust/issues/56192